### PR TITLE
P0 button and alert

### PR DESCRIPTION
**Pull request by Ramandeep9898**

### CODE REVIEW

**Comment by manojsarna** [comment]
1.Overall Text color check contrast. Heading are looking washed out

2. Alerts check color contrast

3. Responsive right left navs are hiding main section.

4. Buttons can have better contrast ratio

5. Current font has limited variety for font weights. Suggestion , please use Google Fonts.

6. can use code snippets for direct access to the particular component.

File: badge.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-s"> 
+                    <span class="badge-span badge-online"></span>
+                </div>
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-md"> 
+                    <span class="badge-span badge-offline"></span>
+                </div>
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-l">
+                    <span class="badge-span badge-dnd"></span>
```
Comment (rahulyadav139) [inline badge.html]: it seems badge fall not on the image. It would be great if it is perfectly set at bottom-left(say) of the image.

Comment (Ramandeep9898) [reply badge.html]: done!!

**Comment by Ramandeep9898** [comment]
> 1.Overall Text color check contrast. Heading are looking washed out
> 
> 2. Alerts check color contrast
> 3. Responsive right left navs are hiding main section.
> 4. Buttons can have better contrast ratio
> 5. Current font has limited variety for font weights. Suggestion , please use Google Fonts.
> 6. can use code snippets for direct access to the particular component.

done!!

**Pull request merged**